### DISCUSSION TITLE
Frigate + Double-take + Codeproject.ai = license plate reading?

### DISCUSSION THREAD

**Discussion by Sceptersax** [skrashevich/double-take#127] [General]
This is great. Thanks for picking this up. I am interested specifically in the ability to detect license plates. I don't see much mentioned here about it, but I figured if Codeproject is involved, maybe the lpr module will be useable. And if not now, maybe down the road.

**Comment by DrSpaldo** [comment]
This would be amazing to see. I love the go2rtc project and Frigate. However, the only viable (easy) way to do licence plate reading at the moment using CodeProjectAI is to have an instance of BlueIris running, so this would be amazing to be part of Double-take!

**Reply by DrSpaldo** [reply]
PS. This is also in the issues https://github.com/skrashevich/double-take/issues/54
